src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,11 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override void ConfineCursor (bool confine)
+        {
+            throw new NotImplementedException ();
```

Comment:
I _think_ the Linux platform is for full screen display drivers (i.e. no window manger like X11). It's mostly intended for things like the Raspberry Pi. I think it would be ok to just always say the cursor is confined for that platform.